runeq/resources/project.py

```diff
@@ -185,3 +502,197 @@ def get_projects(client: Optional[GraphClient] = None) -> ProjectSet:
             break
 
     return project_set
+
+
+def _set_patient_metrics(
+    metrics: dict
+) -> MetricSet:
+    """
+    Helper function that intakes a metric dictionary and outputs a metric set.
+
+    """
+    metric_set = MetricSet()
+
+    for metric in metrics:
+        metric = Metric(**metric)
+        metric_set.add(metric)
+
+    return metric_set
+
+
+def get_project_patients(
+    id: str,
+    client: Optional[GraphClient] = None,
+) -> ProjectPatientSet:
+    """
+    Get all patients in a project and their associated project,
+    processed data metrics.
+
+    Args:
+        id: ID of the project
+        client: If specified, this client is used to fetch metadata from the
+            API. Otherwise, the global GraphClient is used.
+
+    """
+    client = client or global_graph_client()
+    project_patients_query = """
+        query($id: ID, $cursorInput: CursorInput) {
+            project(id: $id) {
+                projectPatientList(
+                    cursorInput: $cursorInput
+                ){
+                    projectPatients{
+                        patient {
+                            id
```

Comment:
Depending on use case, should we return more fields from the `patient` dynamo object?

Reply:
I don't think so. There are queries that return all patient data already. I think the intent here is to return project/cohort specific information.